src/js/jsx/tools/GuidesOverlay.jsx

```diff
@@ -233,6 +242,40 @@ define(function (require, exports, module) {
         },
 
         /**
+         * Returns true if the given D3 component is under the given mouse location
+         *
+         * @private
+         * @param {SVGElement} zone Guide zone being checked
+         * @param {number} x mouse X coordinate
+         * @param {number} y mouse Y coordinate
+         * @return {boolean} True if [x,y] is in the zone
+         */
+        zoneUnderLocation: function (zone, x, y) {
+            var zoneLeft = mathUtil.parseNumber(zone.attr("x")),
+                zoneTop = mathUtil.parseNumber(zone.attr("y")),
+                zoneRight = zoneLeft + mathUtil.parseNumber(zone.attr("width")),
+                zoneBottom = zoneTop + mathUtil.parseNumber(zone.attr("height"));
+            
+            return zoneLeft < x && zoneRight > x &&
```

Comment:
Seems like the edges of the zone should be included? Also, remember `mathUtil.parseNumber` can return null. Presumably if the `zone` were somehow not ready for this, the function will end up returning false? Might want to double-check that.